tests/fixtures/gremlin-console/remote.yaml

```diff
@@ -0,0 +1,9 @@
+hosts: [127.0.0.1]
+username: stephen
```

Comment:
Is there a special significance to this username?

Reply:
Yes, the Tinkerpop graph database comes with a few default users, and it seemed easier to use their pre-provisioned user graph than manually building our own.